terraform/slurm_cluster/modules/slurm_instance_template/main.tf

```diff
@@ -87,7 +87,7 @@ data "local_file" "startup" {
 ############
 
 module "instance_template" {
-  source = "github.com/terraform-google-modules/terraform-google-vm//modules/instance_template?ref=84fbd1f"
+  source = "github.com/terraform-google-modules/terraform-google-vm//modules/instance_template?ref=v11.1.0"
```

Comment:
Sorry, I didn't see this before. The least disruptive update here to still allow for 4.x is:

```suggestion
  source  = "terraform-google-modules/vm/google//modules/instance_template"
  version = "~> 10.1"
```